persistent-sqlite/persistent-sqlite.cabal

```diff
@@ -23,6 +23,18 @@ flag use-pkgconfig
 flag build-sanity-exe
   description: Build a sanity check test executable
   default: False
+flag fulltextsearch
+  description: Enable full-text search
+  default: True
+flag urifilenames
+  description: Enable URI filenames
+  default: True
+flag haveusleep
```

Comment:
Is there any convention for how flags are named in cabal files? I was wondering if there should be a preference for e.g. `have-usleep` instead. So far I'm not sure if there is any convention

Reply:
I'm not sure if there's a convention either =\

How about adding a `cbits-` prefix so it's clear these are cbits related? e.g. `cbits-fulltextsearch`, `cbits-urifilenames`, `cbits-usleep`, `cbits-json1`

Reply:
Nah it's fine. I don't think `cbits` means anything to people consuming persistent-sqlite anyway